operations/pyroscope/helm/pyroscope/templates/configmap-agent.yaml

```diff
@@ -0,0 +1,186 @@
+{{- if .Values.agent.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.agent.agent.configMap.name }}
+  labels:
+    {{- include "pyroscope.labels" . | nindent 4 }}
+data:
+  config.river: |
+    logging {
+    	level  = "info"
+    	format = "logfmt"
+    }
+
+    discovery.kubernetes "pyroscope_kubernetes" {
+    	role = "pod"
+    }
+
+    // The default scrape config allows to define annotations based scraping.
+    //
+    // For example the following annotations:
+    //
+    // ```
+    // profiles.grafana.com/memory.scrape: "true"
+    // profiles.grafana.com/memory.port: "8080"
+    // profiles.grafana.com/cpu.scrape: "true"
+    // profiles.grafana.com/cpu.port: "8080"
+    // profiles.grafana.com/goroutine.scrape: "true"
+    // profiles.grafana.com/goroutine.port: "8080"
+    // ```
+    //
+    // will scrape the `memory`, `cpu` and `goroutine` profiles from the `8080` port of the pod.
+    //
+    // For more information see https://grafana.com/docs/phlare/latest/operators-guide/deploy-kubernetes/#optional-scrape-your-own-workloads-profiles
+    discovery.relabel "kubernetes_pods" {
+    	targets = concat(discovery.kubernetes.pyroscope_kubernetes.targets)
+
+    	rule {
+    		action        = "drop"
+    		source_labels = ["__meta_kubernetes_pod_phase"]
+    		regex         = "Pending|Succeeded|Failed|Completed"
+    	}
+
+    	rule {
+    		action = "labelmap"
+    		regex  = "__meta_kubernetes_pod_label_(.+)"
+    	}
+
+    	rule {
+    		action        = "replace"
+    		source_labels = ["__meta_kubernetes_namespace"]
+    		target_label  = "namespace"
+    	}
+
+    	rule {
+    		action        = "replace"
+    		source_labels = ["__meta_kubernetes_pod_name"]
+    		target_label  = "pod"
+    	}
+
+    	rule {
+    		action        = "replace"
+    		source_labels = ["__meta_kubernetes_pod_container_name"]
+    		target_label  = "container"
+    	}
+    }
+  {{- $profileTypes := list "memory" "cpu" "goroutine" "block" "mutex"  }}
+  {{- range $profileTypes }}
+
+    discovery.relabel "kubernetes_pods_{{.}}_default_name" {
+    	targets = concat(discovery.relabel.kubernetes_pods.output)
```

Comment:
Worth leaving a comment why those need to be tabs, vs. the ones in lines 36-65 are spaces

Reply:
Yeah this is because of the river linter.